proxy/dropCookiesSelectively.js

```diff
@@ -0,0 +1,23 @@
+//@zaproxy-proxy
+
+globalCookies = ['CONSENT', 'GZ', 'SID']
+
+function proxyRequest(msg) {
+    globalCookies.forEach(function (cookieToDrop) {
+        cookies = msg.getRequestHeader().getHttpCookies() // This is a List<HttpCookie>
```

Comment:
This could be called outside the function, to avoid reprocess the cookies each time.